src/utils/sandbox.ts

```
@@ -0,0 +1,46 @@
+    context.require = (moduleName: string) => {
+      if (!allowedModules.includes(moduleName)) {
+        throw new Error(`Module '${moduleName}' is not permitted.`);
+      }
+      return require(moduleName);
+    };
+  }
+
+  const keys = Object.keys(context);
+  const values = Object.values(context);
+
+  const AsyncFunction = Object.getPrototypeOf(async function () {}).constructor;
```

ESLint

[error] Unexpected empty async function.

(@typescript-eslint/no-empty-function)